lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -1079,7 +1079,7 @@ FuncInfo * ByteCodeGenerator::StartBindGlobalStatements(ParseNode *pnode)
         byteCodeFunction->ResetByteCodeGenVisitState();
         if (byteCodeFunction->GetBoundPropertyRecords() == nullptr)
         {
-            Assert(!IsInNonDebugMode());
+            Assert(!IsByteCodeGeneratorInNonDebugMode());
```

Comment:
Nit: useless assert - there is an if condition above which checks the same condition.